src/modules/item/sagas.ts

```diff
@@ -360,10 +360,9 @@ export function* itemSaga(legacyBuilder: LegacyBuilderAPI, builder: BuilderClien
     const isEmotesFeatureFlagOn: boolean = yield select(getIsEmotesFlowEnabled)
     const { item } = action.payload
     const collectionId = item.collectionId!
-    if (openModals['EditItemURNModal']) {
-      yield put(closeModal('EditItemURNModal'))
-    } else if (openModals['EditPriceAndBeneficiaryModal']) {
-      yield put(closeModal('EditPriceAndBeneficiaryModal'))
+    const ItemModals = ['EditItemURNModal', 'EditPriceAndBeneficiaryModal', 'AddExistingItemModal']
```

Comment:
This change is not related to this issue. It closes all opened modals when creating/updating an item.